private/bufpkg/bufconnect/interceptors_test.go

```diff
@@ -25,14 +25,32 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestAuthorizationInterceptorProviderTokenErr(t *testing.T) {
-	_, err := NewAuthorizationInterceptorProvider(app.NewEnvContainer(map[string]string{
-		tokenEnvKey: "test1234",
-	}))("fake")(func(context.Context, connect.AnyRequest) (connect.AnyResponse, error) {
-		return nil, errors.New("underlying cause")
+func TestNewAuthorizationInterceptorProvider(t *testing.T) {
+	tokenSet, err := NewTokenSetFromString("user1:token1@remote1,token")
+	assert.NoError(t, err)
+	_, err = NewAuthorizationInterceptorProvider(tokenSet)("default")(func(ctx context.Context, req connect.AnyRequest) (connect.AnyResponse, error) {
+		if req.Header().Get(AuthenticationHeader) != AuthenticationTokenPrefix+"token" {
+			return nil, errors.New("error auth token")
+		}
+		return nil, nil
 	})(context.Background(), connect.NewRequest(&bytes.Buffer{}))
+	assert.NoError(t, err)
+}
+
+func TestNewTokenSetFromEnv(t *testing.T) {
+	_, err := NewTokenSetFromContainer(app.NewEnvContainer(map[string]string{}))
+	assert.NoError(t, err)
```

Comment:
Check for values here too, not just error.

Reply:
Now also checking the token value by looking up from remotes.